index.d.ts

```diff
@@ -1,48 +1,24 @@
-import * as APITypes from './typescript/api'
-import * as FormTypes from './typescript/forms'
-import * as MiscTypes from './typescript/misc'
-import * as SubmissionEventTypes from './typescript/submissionEvents'
-import * as FormsAppsTypes from './typescript/formsApps'
-import * as EnvironmentTypes from './typescript/environments'
-import * as SubmissionTypes from './typescript/submissions'
-import * as KeyTypes from './typescript/keys'
-import * as IntegrationTypes from './typescript/integrations'
-import * as TeamMemberTypes from './typescript/teamMembers'
-import * as OrganisationTypes from './typescript/organisations'
-import * as GeoscapeTypes from './typescript/geoscape'
-import * as PointTypes from './typescript/point'
-import * as CivicaTypes from './typescript/civica'
-import * as AWSTypes from './typescript/aws'
-import * as ApprovalTypes from './typescript/approvals'
-import * as ConditionTypes from './typescript/conditions'
-import * as PDFTypes from './typescript/pdf'
-import * as SchedulingTypes from './typescript/scheduling'
-import * as CDNTypes from './typescript/cdn'
-import * as EmailTemplateTypes from './typescript/emailTemplates'
-import * as FreshdeskTypes from './typescript/freshdesk'
-import * as PDFConversionTypes from './typescript/pdfConversion'
-export {
-  APITypes,
-  FormTypes,
-  MiscTypes,
-  SubmissionEventTypes,
-  FormsAppsTypes,
-  EnvironmentTypes,
-  SubmissionTypes,
-  KeyTypes,
-  IntegrationTypes,
-  TeamMemberTypes,
-  OrganisationTypes,
-  GeoscapeTypes,
-  PointTypes,
-  CivicaTypes,
-  AWSTypes,
-  ApprovalTypes,
-  ConditionTypes,
-  PDFTypes,
-  SchedulingTypes,
-  CDNTypes,
-  EmailTemplateTypes,
-  FreshdeskTypes,
-  PDFConversionTypes,
-}
```

Comment:
Isn't this going to break everything that uses `FormTypes.Form` for example?

Reply:
Because `FormTypes` won't exist anymore, `Form` will just be exported from types

Reply:
`FormTypes` is being exported on line 2

Reply:
Oh yeh, I misread is as exporting all from.

Reply:
not as.